README.md

```diff
@@ -443,32 +443,72 @@ of the time a higher-level object will suffice as an argument.
 **Bad:**
 
 ```php
-function createMenu(string $title, string $body, string $buttonText, bool $cancellable): void
-{
-    // ...
+class Questionnaire
+{
+    public function __construct(
+        string $firstname,
+        string $lastname,
+        string $patronymic,
+        string $region,
+        string $district,
+        string $city,
+        string $phone,
+        string $email
+    ) {
+        // ...
+    }
 }
 ```
 
 **Good:**
 
 ```php
-class MenuConfig
+class Name
 {
-    public $title;
-    public $body;
-    public $buttonText;
-    public $cancellable = false;
+    private $firstname = '';
+    private $lastname = '';
+    private $patronymic = '';
+
+    public function __construct(string $firstname, string $lastname, string $patronymic)
+    {
+        $this->firstname = $firstname;
+        $this->lastname = $lastname;
+        $this->patronymic = $patronymic;
+    }
```

Comment:
Add getters or add a comment to explain that there will be some getters to retrieve those data.

Reply:
I add comment for this